modules/cluster-logging-exported-fields-kubernetes.adoc

```diff
@@ -1,60 +1,97 @@
-// Module included in the following assemblies:
-//
-// * logging/cluster-logging-exported-fields.adoc
-
 [id="cluster-logging-exported-fields-kubernetes_{context}"]
-= Kubernetes exported fields
+== Kubernetes
+
+// Normally, the preceding title would be an H1 prefixed with an `=`. However, because the following content is auto-generated at https://github.com/ViaQ/documentation/blob/main/src/data_model/public/kubernetes.part.adoc and pasted here, it is more efficient to use it as-is with no modifications. Therefore, to "realign" the content, I am going to prefix the title with `==` and use `include::modules/cluster-logging-exported-fields-kubernetes.adoc[leveloffset=0]` in the assembly file.
+
+// DO NOT MODIFY THE FOLLOWING CONTENT. Instead, update https://github.com/ViaQ/documentation/blob/main/src/data_model/model/kubernetes.yaml and run `make` as instructed here: https://github.com/ViaQ/documentation
+
+The namespace for Kubernetes-specific metadata. All these fields are optional.
```

Comment:
@alanconway I added this line plus "Optional." to each of the following descriptions.